evalai/utils/challenges.py

```diff
@@ -1,10 +1,10 @@
 import json
 import requests
 import sys
-
+from termcolor import colored
```

Comment:
@RohanSreelesh please move this to last import after `datetime`. We follow alphabetical order for imports